packages/core/test/unit/html/NodeProcessor.data.ts

```diff
@@ -230,16 +230,7 @@ export const PROCESS_MODAL_OK_TEXT_EXPECTED = `
 </modal>
 `;
 
-// todo remove these once modal-header modal-footer slot names are deprecated fully.
-
-export const PROCESS_MODAL_SLOTS_RENAMING = `
-<modal>
-  <div slot="modal-header">Should be renamed to header</div>
-  <div slot="modal-footer">Should be renamed to footer</div>
-</modal>
-`;
-
-export const PROCESS_MODAL_SLOTS_RENAMING_EXPECTED = `
+module.exports.PROCESS_MODAL_SLOTS_RENAMING_EXPECTED = `
```

Comment:
This should be removed as well - it's the expected output of the removed test case